samples/excel/10-chart/chart-axis-formatting.yaml

```diff
@@ -52,8 +52,10 @@ script:
 
         async function setup() {
             await Excel.run(async (context) => {
-                let sheet = await OfficeHelpers.ExcelUtilities
-                    .forceCreateSheet(context.workbook, "Sample");
+                context.workbook.worksheets.getItemOrNullObject("Sample").delete();
```

Comment:
With here and anywhere you're removing "forceCreateSheet":  are you sure that the sample would only have worked on ExcelApi 1.4 and higher?

If so, great.  But I thought some samples (e.g., report generation) were supposed to work on older (RTM) versions of Office 2016 as well.  Whereas by you making use of the "OrNullObject" functionality, you are effectively breaking that.

See https://github.com/OfficeDev/office-js-helpers/blob/master/src/excel/utilities.ts#L103

Reply:
Office 2016 only supports ExcelApi 1.1, so most of the samples don't work there anyway. Since everything else should be relatively up to date, I'm not sure there's value in targeting 1.2-1.4 era builds.

Plus, we've never called out required versions in SL snippets. Perhaps that's a valid thing to do in the future, but I feel like it's out of scope here.

Reply:
@AlexJerabek , each sample has an API set at the top of the file in the YAML, which Script Lab does use.  E.g,.:

```
api_set:
    ExcelAPI: 1.8
```

So, what I'm saying is that for any sample where you've removed "forceCreateSheet", you should at minimum make sure that the API set of that sample was >= 1.4, and/or update it to be greater than 1.4.

When you do, for the report generation sample, you can also get rid of some of the other "isSetSupported" checks within it, since I believe it checks for 1.3 req set -- which won't be relevant once you make the whole sample require 1.4

Reply:
BTW, I think it would also be interesting (and easy?) to look at the OMEX telemetry to see the distribution of build numbers for people using Script Lab.  I.e., is it true that 90% (or whatever) of SL users are on reasonably recent builds?  Or is there still a non-trivial contingent of people who are on 2016 MSI?..